cli/chain.go

```diff
@@ -63,6 +68,8 @@ var ChainCmd = &cli.Command{
 		ChainEncodeCmd,
 		ChainDisputeSetCmd,
 		ChainPruneCmd,
+		ChainExecEVMCmd,
```

Comment:
`chain` is not the right place for this, can they go into the new `eth` commands?

Reply:
And rename execevm to create-evm or some such.

Reply:
They're now in the `evm` command group.